README.md

```diff
@@ -78,8 +87,13 @@ If you just want to run the system, see "Run the Code" below.
    Note that this script is just a convenience to install system-wide dependencies we expect.
    As a result, it uses the system package manager, requires `sudo`, and should only be run **once**.
    ```console
-   # ./scripts/install-build-tools.sh
+   $ sudo ./scripts/install-build-tools.sh
```

Comment:
I mentioned that specifying `sudo` here didn't make sense because we had `#` already; I suppose one valid way of fulfilling that was to use `$` and specify `sudo`, but I'm unclear on why that would be preferred instead of dropping `sudo` in favor of the `#`.

Is there a reason to prefer specifying `sudo` (which not everyone uses)?